rfcs/text/0030-linux-event-model.md

```diff
@@ -0,0 +1,1121 @@
+ * Ingestion mechanisms (e.g. beats/logstash)
+ * Usage mechanisms (e.g. Kibana applications, detections)
+ * ECS project (e.g. docs, tooling)
+The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
+-->
+
+## Concerns
+
+<!--
+Stage 1: Identify potential concerns, implementation challenges, or complexity. Spend some time on this. Play devil's advocate. Try to identify the sort of non-obvious challenges that tend to surface later. The goal here is to surface risks early, allow everyone the time to work through them, and ultimately document resolution for posterity's sake.
+-->
+
+Potential concerns for me are primary tied to message size. The nested ancestors and their many fields could add up. That being said, there is a larger concern wrt number of events that will dwarf any issue around message size. That effort will come down to defining a set of event filters to eliminate unwanted noise. Scraping /proc and sending up all process executions can be very load heavy task. This issue is probably outside of the scope of ECS though.
+
+Another point of concern is the re-use of process.entity_id. There is nuance around how this ID is calculated, and we are proposing a new hash/calculation based of a different set of fields. This issue is being tracked here: https://github.com/elastic/security-team/issues/2458
+
+<!--
+Stage 2: Document new concerns or resolutions to previously listed concerns. It's not critical that all concerns have resolutions at this point, but it would be helpful if resolutions were taking shape for the most significant concerns.
+-->
+
+<!--
+Stage 3: Document resolutions for all existing concerns. Any new concerns should be documented along with their resolution. The goal here is to eliminate risk of churn and instability by ensuring all concerns have been addressed.
+-->
+
+## People
+
+The following are the people that consulted on the contents of this RFC.
+
+* @mitodrummer | author
+* @mitodrummer & @m-sample | sponsor
+* @m-sample | subject matter expert
+* @norrietaylor | subject matter expert
+
+<!--
+Who will be or has been consulted on the contents of this RFC? Identify authorship and sponsorship, and optionally identify the nature of involvement of others. Link to GitHub aliases where possible. This list will likely change or grow stage after stage.
+
+e.g.:
+
+* @Yasmina | author
+* @Monique | sponsor
+* @EunJung | subject matter expert
+* @JaneDoe | grammar, spelling, prose
+* @Mariana
+-->
+
+
+## References
+
+<!-- Insert any links appropriate to this RFC in this section. -->
+
+https://github.com/elastic/security-team/blob/main/docs/adaptive-workload-protection-team/architecture/linux-event-model.md
+https://github.com/elastic/security-team/pull/2071/files
```

Comment:
Can y'all pull this information out into a public issue or otherwise summarize it? Most people reading this RFC won't be able to access this content as-is.

Reply:
@m-sample Were there snippets of your doc we could include inline here, or could we put some of it into a public issue?

Reply:
Yes, I will create a condensed version we can include.